lib/charms/prometheus_k8s/v0/prometheus_remote_write.py

```diff
@@ -1043,7 +1036,7 @@ def alerts(self) -> dict:
         """
         alerts = {}  # type: Dict[str, dict] # mapping b/w juju identifiers and alert rule files
         for relation in self._charm.model.relations[self._relation_name]:
-            if not relation.units:
+            if not relation.units or not relation.app:
```

Comment:
I guess this may work as is but I wonder if we should check app first and then unit.

Reply:
This is a Prometheus Lib, not a Grafana agent one.